docs/source/using/command_line.rst

```diff
@@ -1,28 +1,30 @@
 Using the Command Line
 ======================
 
+.. _cli-ref:
+
 Both the BDSS and the Workflow Manager can be invoked from the command line whilst
-running a bash shell in the  ``force-py36`` Enthought Deployment Manager (EDM)
-environment.
+running a bash shell in the :ref:`force-pyXX environment  <bdss-environment-ref>`.
```

Comment:
Although I think using the generic `force-pyXX` terminology in the installation documentation is good because avoids us needing to update it with python version, we might want to pick a concrete environment name (`force-py36`) for this section just to avoid confusion.

We can state "here we demonstrate how to invoke the `force_bdss` and `force_wfmanager` from an example development environment, called `force-py36`", to make things clearer.